gulpfile.js

```diff
@@ -11,12 +11,15 @@ const gulp      = require('gulp'),
   uglify        = require('gulp-uglify'),
   useref        = require('gulp-useref'),
   plumber       = require('gulp-plumber'),
-  imagemin      = require('gulp-imagemin')
+  imagemin      = require('gulp-imagemin'),
   size          = require('gulp-filesize'),
   gulpStylelint = require('gulp-stylelint'),
   minifyCSS     = require('gulp-minify-css'),
   autoprefixer  = require('gulp-autoprefixer'),
-  browserSync   = require('browser-sync').create();
+  browserSync   = require('browser-sync').create(),
+  shell = require('gulp-shell');
+
+gulp.task('generate-styleguid', shell.task('styledown src/styleguid/config.md > src/styleguid/styleguide.html'))
```

Comment:
I would like to put it in a `public` folder at the root of the project so
- src
- scripts
- public

Reply:
fixed